python/servo/package_commands.py

```diff
@@ -105,6 +105,32 @@ def copy_dependencies(binary_path, lib_path):
         need_checked.difference_update(checked)
 
 
+def change_prefs(resources_path, platform):
+    prefs_path = path.join(resources_path, "prefs.json")
+    package_prefs_path = path.join(resources_path, "package-prefs.json")
+    os_type = "os:{}".format(platform)
+    with open(prefs_path) as prefs:
+        prefs = json.loads(prefs.read())
+        with open(package_prefs_path) as package_prefs:
+            package_prefs = json.loads(package_prefs.read())
+            for pref in package_prefs:
+                if os_type in pref:
+                    prefs[pref.split(";")[1]] = package_prefs[pref]
+                if pref in prefs:
+                    prefs[pref] = package_prefs[pref]
+            with open(prefs_path, "w") as out:
+                json.dump(prefs, out, sort_keys=True, indent=2)
+                out.close()
```

Comment:
I don't think we need to explicitly close this (since we're inside a context manager)?